fuzz/Cargo.toml

```diff
@@ -33,7 +33,7 @@ quote = "1.0"
 component-fuzz-util = { path = "../crates/misc/component-fuzz-util" }
 
 [features]
-default = ['fuzz-spec-interpreter']
+default = []
```

Comment:
This change looks like it was accidentally committed.

Reply:
Urgh, yep, good catch, and thanks!